disttask/loaddata/proto.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package loaddata
+
+import (
+	"github.com/pingcap/tidb/br/pkg/lightning/backend"
+	"github.com/pingcap/tidb/br/pkg/lightning/config"
+	"github.com/pingcap/tidb/br/pkg/lightning/mydump"
+	"github.com/pingcap/tidb/parser/model"
+	"github.com/pingcap/tidb/parser/mysql"
+)
+
+// TaskStep of LoadData.
+const (
+	ReadSortImport = 1
+)
+
+// Task is the task of LoadData.
+type Task struct {
+	Table     Table
+	Format    Format
+	Dir       string
+	FileInfos []FileInfo
+}
+
+// Subtask is the subtask of LoadData.
+type Subtask struct {
+	Table  Table
+	Format Format
+	Dir    string
+	Chunks []Chunk
+}
+
+// MinimalTask is the minimal task of LoadData.
+type MinimalTask struct {
+	Table  Table
+	Format Format
+	Dir    string
+	Chunk  Chunk
+	Writer *backend.LocalEngineWriter
+}
+
+// IsMinimalTask implements the MinimalTask interface.
+func (MinimalTask) IsMinimalTask() {}
+
+// Table records the table information.
+type Table struct {
```

Comment:
In future I hope they are replaced by the load data plan
https://github.com/pingcap/tidb/blob/3c942f7e101d6b0aff85c552598678f2a761deca/planner/core/common_plans.go#L547-L565